bundle/src/test/java/com/adobe/acs/commons/properties/filter/TemplatedFilterTest.java

```diff
@@ -0,0 +1,272 @@
+
+        FilterChain mocked = mock(MockFilterChain.class);
+        String before = "{\"jcr:primaryType\":\"nt:unstructured\",\"fileReference\":\"/content/dam/we-retail/en/experiences/arctic-surfing-in-lofoten/surfer-wave-01.jpg\",\"useFullWidth\":\"true\",\"stringArray\":[\"{{page_properties.pageTitle}}\",\"{{page_properties.jcr:title}}\"],\"objectArray\":[{\"pageTitle\":\"{{page_properties.pageTitle}}\"},{\"jcrTitle\":\"{{page_properties.jcr:title}}\"}],\"sling:resourceType\":\"weretail/components/content/heroimage\"}";
+        doAnswer(invocation -> {
+            HttpServletResponse response =
+                    (HttpServletResponse) invocation.getArguments()[1];
+            response.setCharacterEncoding("utf-8");
+            response.setContentType("application/json");
+            response.getWriter().println(before);
+            return null;
+        }).when(mocked).doFilter(any(), any());
+
+        String after = "{\"jcr:primaryType\":\"nt:unstructured\",\"fileReference\":\"/content/dam/we-retail/en/experiences/arctic-surfing-in-lofoten/surfer-wave-01.jpg\",\"useFullWidth\":\"true\",\"stringArray\":[\"Surfing In Arctic Lofoten\",\"Arctic Surfing In Lofoten\"],\"objectArray\":[{\"pageTitle\":\"Surfing In Arctic Lofoten\"},{\"jcrTitle\":\"Arctic Surfing In Lofoten\"}],\"sling:resourceType\":\"weretail/components/content/heroimage\"}";
+        filter.doFilter(context.request(), context.response(), mocked);
+        assertTrue(startsWith(after)
+                .matches(context.response().getOutputAsString()));
+    }
+
+    @Test
+    public void testInvalidJson() throws IOException, ServletException {
+        initServices(null);
+        context.currentResource("/content/we-retail/language-masters/en/experience/arctic-surfing-in-lofoten/jcr:content/root/hero_image");
+        context.request().setResource(context.currentResource());
+        context.request().setPathInfo("/content/we-retail/language-masters/en/experience/arctic-surfing-in-lofoten/jcr:content/root/hero_image.model.json");
+        context.requestPathInfo().setExtension("json");
+
+        FilterChain mocked = mock(MockFilterChain.class);
+        String before = "<html><body>{{page_properties.jcr:title}}</body></html>";
+        doAnswer(invocation -> {
+            HttpServletResponse response =
+                    (HttpServletResponse) invocation.getArguments()[1];
+            response.setCharacterEncoding("utf-8");
+            response.getWriter().println(before);
+            return null;
+        }).when(mocked).doFilter(any(), any());
+
+        filter.doFilter(context.request(), context.response(), mocked);
+        assertTrue(startsWith(before)
+                .matches(context.response().getOutputAsString()));
+    }
+
+    @Test
+    public void testExcludeConfig() throws IOException, ServletException {
+        Map<String, Object> config = new HashMap<>();
+        config.put("excludes", new String[]{"(.*).model.(.*)"});
```

Comment:
Should we update this matcher to what we updated to in the code?

Reply:
updated